organization/annual-reports/financial-report-2021.md

```diff
@@ -0,0 +1,557 @@
+---
+type: resource
+---
+
+# Financial annual report: 1 January 2021 - 31 December 2021
+
+Foundation for Public Code (Vereniging)
+
+## A. Narrative
+
+The Foundation for Public Code was founded by Ben Cerveny, Boris van Hoytema and Arnout Schuijff in 2019 to guide international collaboration among public organizations around codebase design, quality and governance.
+
+Leveraging 30 combined years of experience working with cities and technology-enabled collaboration, Boris and Ben recognized policy and whitepapers were ineffective catalysts in this space. Their landscape analysis revealed that stewardship was the key to converting the massive public investment in open source and collaboration into long-term, sustainable implementation.
+
+Based on their experience of technological innovation in a traditional industry, Pieter van der Does and Arnout Schuijff (co-founders of the Adyen payments platform) agreed with Ben and Boris that public organizations deserved better software. They joined the Board of Directors to share their expertise and connections.
+
+### 1. General
+
+#### 1.1 Establishment and purpose; mission and vision
+
+##### 1.1.1 Establishment
+
+The Foundation for Public Code was established on June 3, 2019 and is registered in the Commercial Register of the Chamber of Commerce under number 74996452. The statutory name of the organization is: Foundation for Public Code Vereniging (Association) and has its registered office in Amsterdam.
+
+##### 1.1.2 Statutory goal
+
+The goal of the Foundation for Public Code is to support collective development and maintenance of public code (software and policy with a public purpose) in order to reduce risks, increase quality, reduce costs and increase sustainability (article 3.1 of the association bylaws).
+
+Public code stewarded by the Foundation is openly available for reuse by everyone, but only members (wholly publicly owned organizations) can determine the organization’s governance and the stewardship priorities.
+
+The association seeks to achieve its goal by carrying out the following activities (article 3.2 of the association bylaws):
+
+* 1. the provision of services that can increase sustainability, quality and re-use of public code
+* 2. facilitating and encouraging the sharing of existing public code between public members and other public organizations
+* 3. carrying out research for the (further) development of public code
+* 4. drawing attention to problems ascertained and giving advice with regard to the public code they use
+* 5. everything related to or conducive to the above, all in the broadest sense
```

Comment:
I don't think we want these to be both bullets and numbers

Reply:
This is a [straight copy from our articles of association](https://about.publiccode.net/organization/articles-of-association.html#article-3-purpose-and-methodology), including the formatting.

Reply:
Sorry, I'm wrong. I'll play around with it.